### ISSUE TITLE
3.25.0 release did not get completed

### ISSUE THREAD

**Issue by chenrui333** [kevinburke/go-bindata#52]
👋 it looks like the 3.25.0 release was not listed as latest release, just create this issue to confirm if that is intended latest release. Thanks!

relates to https://github.com/Homebrew/homebrew-core/pull/126295

**Comment by kevinburke** [comment]
I will fix once I put kid to bed sorry

**Comment by chenrui333** [comment]
no rush at all, and appreciate the fix!

**Comment by chenrui333** [comment]
looks like there is v4 tag besides v4.0.0, not sure if that is intentional :)
![image](https://user-images.githubusercontent.com/1580956/227569414-c3eb33ef-cc31-4cd1-83d8-80bb8f57c356.png)

**Comment by kevinburke** [comment]
Yeah I am trying to get "go get ...@latest" working and don't have it
working quite yet. Hang on one sec

--
Kevin Burke
phone: 925-271-7005 | kevin.burke.dev
